assets/_sass/_navbar.scss

```diff
@@ -41,7 +41,7 @@ $nav-item-border: darken($navbar-default-bg, 8%);
   position: sticky;
   border-right: 0.5px solid $color-grey-3;
   top: 0;
-  height: calc(100vh - 50px);
+  height: 100vh;
```

Comment:
Does this not cause an overflow over the top nav?

Reply:
Nope, that gap was on the bottom, removed it now. It looks like such:
![image](https://github.com/user-attachments/assets/89f400d4-b3c4-4994-b90b-467be4df834e)

Also removes the gap on `hugo serve` now

Reply:
![Screenshot 2024-07-18 at 15 19 47](https://github.com/user-attachments/assets/f86bfae0-4030-429c-b527-225835f2481a)

My side the footer is now massive with removing this.

Reply:
Will address this on the apple site as this is an apple issue 🙁